packages/bimage-gtk/bimage-gtk.0.1.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Zach Shipko <zachshipko@gmail.com>"
+authors: ["Zach Shipko <zachshipko@gmail.com>"]
+homepage: "https://github.com/zshipko/ocaml-bimage"
+doc: "https://zshipko.github.io/ocaml-bimage/doc"
```

Comment:
For the record this page doesn't exist

Reply:
Thanks, I've just updated the docs as well.